application/helpers/expressions/em_manager_helper.php

```diff
@@ -6607,11 +6606,41 @@ function _ValidateQuestion($questionSeq,$force=false)
                     }
                 }
             }
-
+            /**
+             * Control value against value from survey : see #11611
+             */
+            if ($qrel)
```

Comment:
At start , i completely add a new function to make validation but .... in fact ProcessCurrentResponse is done for this : validation responses.

I can create a new function again with an hundred  var .... (here 

````
foreach($sgqas as $sgqa)
{
  if($this->getValidityPost($qid,$sqga,$qhidden,....);
  {
    $stringToParse .= Yii::app()->getController()->renderPartial('/survey/system/questionhelp/tips', array('qid'=>$qid,'vclass'=>'dberror alert alert-danger','vtip'  =>$this->invalidAnswerCore[$sgqa]), true);
    $qvalid=false;
  }
}
````
because actual function need $stringToParse and $qvalid to be updated.

And thid need to be done in $sgqa.

You ask me to rewrite completely _ValidateQuestion .